### ISSUE TITLE
Add Fullbody Leg and HIp to Non avatar setup with it.

### ISSUE THREAD

**Issue by Jazzneo** [Frooxius/NeosPublic#303]
Some people who try fullbody will find some avatar not setup with fullbody would be possible add feature to avatar creator let you add fullbody hip and leg on premade avatars?

**Comment by Frooxius** [comment]
Yes, the full body is currently work in progress, next thing it will have is making it calibrate your avatar as well, so that will allow us to easily recalibrate all existing avatars for full body.
